kontrakter/vegtrafikkulykke/trafikkulykke/1.0/trafikkuhell.schema.json

```diff
@@ -0,0 +1,637 @@
+        "enhetsbokstav": {
+          "type": "string",
+          "description": "Enheten sin ID i saken. A, B, C...."
+        },
+        "registreringsnummer": {
+          "type": "string"
+        },
+        "enhetstype": {
+          "$ref": "#/definitions/enhetstype"
+        },
+        "aarsmodell": {
+          "type": "integer"
+        },
+        "registreringsland": {
+          "type": "string",
+          "description": "ISO 3166-1-alpha-3 landkode"
+        },
+        "dekktype": {
+          "$ref": "#/definitions/dekktype"
+        },
+        "kommentarDekk": {
+          "type": "string"
+        },
+        "regulerTrafikkenhetVegkryss": {
+          "$ref": "#/definitions/regulerTrafikkenhetVegkryss"
+        },
+        "person/trafikant": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/person"
+          },
+          "description": "Informasjon om person involvert i ulykken."
+        }
+      },
+      "required": ["id"],
+      "additionalProperties": false
+    },
+    "plasseringKjoeretoey": {
+      "type": "string",
+      "enum": ["FOERERPLASS", "FORAN", "BAK", "ANNET"]
+    },
+    "skadegrad": {
+      "type": "string",
+      "enum": [
+        "DREPT",
+        "MEGET_ALVORLIG_SKADD",
+        "ALVORLIG_SKADD",
+        "LETTERE_SKADD",
+        "USKADD",
+        "ILLEBEFINNENDE_FOERER",
+        "SELVDRAP",
+        "IKKE_KONTROLLERT"
+      ]
+    },
+    "beskyttelsesutstyr": {
+      "type": "string",
+      "enum": [
+        "INGEN_BESKYTTELSE",
+        "BILBELTE",
+        "KOLLISJONSPUTE",
+        "BILBELTE_OG_KOLLISJONSPUTE",
+        "BARNESETE",
+        "HJELM",
+        "REFLEKS_ELLER_LYKT",
+        "ANNEN_BESKYTTELSE",
+        "UKJENT"
+      ]
+    },
+    "skadeforloep": {
+      "type": "string",
+      "enum": [
+        "INGEN_PERSONSKADE",
+        "SKADD_MOT_INTERIOER",
+        "KASTET_UT_AV_KJOERETOEY",
+        "PAAKJOERT_FOTGJENGER_AKENDE_ELLER_LIGNENDE",
+        "BRANN",
+        "DRUKNING",
+        "SKADD_MOT_EKSTERIOER",
+        "NAKKESKADE",
+        "ANNET",
+        "UKJENT"
+      ]
+    },
+    "ferdselsformaal": {
+      "type": "string",
+      "enum": [
+        "TIL_FRA_ARBEID",
+        "I_ARBEID",
+        "TIL_FRA_SKOLE",
+        "TIL_FRA_BUTIKK_BANK_ETC",
+        "FRITIDSREISE",
+        "OEVELSESKJOERING",
+        "LEK",
+        "HJEM_PAA_PERMISJON",
+        "TIL_FRA_FEST",
+        "ANNET_FORMAAL",
+        "UKJENT"
+      ]
+    },
+    "person": {
+      "type": "object",
+      "description": "Informasjon om person involvert i ulykken",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gjør personen unikt identifiserbar innad i saken, må være lik ved oppdateringsmelding."
+        },
+        "kjoenn": {
+          "type": "string",
+          "enum": ["MANN", "KVINNE"]
+        },
+        "foedselsdato": {
+          "type": "string",
+          "format": "date"
```

Comment:
Vi kan jo legge inn en beskrivelse om format, eks. [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)

Reply:
Kan det - men det er og implisitt gitt når man spesifiserer format `date`:

>This SHOULD be a date in the format of YYYY-MM-DD. It is recommended that you use the "date-time" format instead of "date" unless you need to transfer only the date part.

https://datatracker.ietf.org/doc/html/draft-zyp-json-schema-03#section-5.23